lizmap/modules/view/locales/en_US/dictionnary.UTF-8.properties

```
@@ -56,6 +56,7 @@ edition.confirm.cancel=Are you sure you want to cancel the edition? You will los
 edition.confirm.restart-drawing=Are you sure you want to delete the existing geometry and redraw it from scratch?
 edition.confirm.paste=Are you sure you want to replace the existing geometry with the one in the clipboard?
 edition.geom.paste=Paste the geometry
+edition.message.error.fetch.form=Form fetching error!
```

```suggestion
edition.message.error.fetch.form=Error while fetching the editing form for that layer. Please contact the system administrator.
```

I don't think you should use `!` in error message. It's the not the fault of the end user.